pkg/mock/mock_test.go

```diff
@@ -59,15 +64,51 @@ func TestMock_IsMockEnabled(t *testing.T) {
 				}),
 				namespace: "default",
 			},
+			args: args{
+				license: &kotsv1beta1.License{
+					Spec: kotsv1beta1.LicenseSpec{
+						LicenseType: "dev",
+					},
+				},
+			},
 			want:    true,
 			wantErr: false,
 		},
+		{
+			name: "not enabled because not a dev license",
+			fields: fields{
+				clientset: fake.NewSimpleClientset(&corev1.SecretList{
+					TypeMeta: metav1.TypeMeta{},
+					ListMeta: metav1.ListMeta{},
+					Items: []corev1.Secret{{
+						TypeMeta: metav1.TypeMeta{},
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      replicatedSecretName,
+							Namespace: "default",
+						},
+						Data: map[string][]byte{
+							replicatedMockEnabledKey: []byte("true"),
+						},
+					}},
+				}),
+				namespace: "default",
+			},
+			args: args{
+				license: &kotsv1beta1.License{
+					Spec: kotsv1beta1.LicenseSpec{
+						LicenseType: "paid",
+					},
+				},
+			},
+			want:    false,
+			wantErr: false,
+		},
```

Comment:
should we add a test case where the license type is `dev`, but the secret contains a `REPLCATED_MOCK_ENABLED: "false"` to test that dev licenses can override?